site/docs/master/restic.md

```diff
@@ -298,7 +298,11 @@ data:
   # If not set, it will default to "128Mi". A value of "0" is treated as unbounded.
   memLimit: 128Mi
 
+  # "secCtxRunAsUser sets the securityContext.runAsUser value on the restic init containers during restore."
```

Comment:
This file has been moved to `site/docs/main/restic.md`. This is what is flagged as a merge conflict.
Can you please resolve this merge conflict?